ComposeSample/app/src/test/java/com/microsoft/device/display/samples/composesample/ExampleUnitTest.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ *  * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.composesample
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
If you are not planning to add unit tests now, you can get rid of this class as it just adds noise that we have to maintain.